user_docs/en/changes.t2t

```diff
@@ -39,7 +39,9 @@ Windows 8.1 is the minimum Windows version supported.
 - A new Native Selection mode (toggled by ``NVDA+shift+f10``) is now available in NVDA's browse mode for Mozilla Firefox.
 When turned on, selecting text in browse mode will also manipulate Firefox's own native selection.
 Copying text with ``control+c`` will pass straight through to Firefox, thus copying the rich content, rather than NVDA's plain text representation. (#15830)
+Note however  that as Firefox is handling the actual copy, NVDA will not report a "copy to clipboard" message in this mode.
 - When copying text in Microsoft Word with NVDA's browse mode enabled, formatting is now also included. (#16129)
+A side affect of this is that NvDA will no longer report  a "copy to clipboard" message when pressing `control+c` in Microsoft Word / Outlook browse mode, as the application is now handling the copy, not NVDA. 

```

Comment:
could you please update these items to move the issue number to the end of the change log item